package.json

```diff
@@ -33,11 +34,15 @@
   "engines": {
     "node": ">= 4.0.0"
   },
-  "files": ["dist", "index.d.ts"],
+  "files": [
+    "dist",
+    "index.d.ts"
+  ],
   "devDependencies": {
+    "@babel/preset-env": "^7.0.0-beta.4",
     "babel-cli": "^6.22.2",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-es2015": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
```

Comment:
Unless I'm mistaken, we only need one of either `babel-preset-env` or `@babel/preset-env`..